src/Plant/photosynthesis/get_leaf_an_ag_r_from_pi.jl

```diff
@@ -0,0 +1,40 @@
+# get a from ci
+function get_leaf_an_ag_r_from_pi(;
+                                  v25::Number = 80.0,
```

Comment:
Please follow the convention in ClimateMachine.jl, where `FT` is used to indicate the float type.
```suggestion
                                  v25::FT = FT(80.0),
```